src/confirmation-dialog/ConfirmationDialog.jsx

```diff
@@ -61,4 +55,12 @@ class ConfirmationDialog extends React.Component {
     }
 }
 
+function renderNode(node) {
+    if (typeof node === "string") {
```

Comment:
When I needed to display non-string nodes as part of the "description" I normally sent them as ``children`` wrapped with a ``DialogContent``.

The change as-is looks good to me, but if we want to avoid the ``typeof`` checking for ``description``, that could still be an alternative.

Reply:
>  I normally sent them as children

Exactly my thoughts, and in fact we are already rendering the children in a DialogContent, just that we had a post-processing on the description (split to use html paragraphs), so I didn't found another way of not breaking existing code.